### PR TITLE
Fixed the footer

### PR DESCRIPTION
**Pull request by mahimagoyalx**
Issue #104

### CODE REVIEW

**Comment by mahimagoyalx** [comment]
Please review. @saviourcode

**Comment by saviourcode** [comment]
pr107
@mahimagoyalx I could see the content horizontal overflow for Screen-width of 771px up to 829px. Kindly fix this horizontal overflow

**Comment by mahimagoyalx** [comment]
> pr107
> @mahimagoyalx I could see the content horizontal overflow for Screen-width of 771px up to 829px. Kindly fix this horizontal overflow

Okay

**Comment by mahimagoyalx** [comment]
Please check now @saviourcode

File: index.html [hunk 1]
```diff
@@ -354,19 +349,21 @@ <h4>Follow Us</h4>
             </div>
           </div>
         </div>
-        <div>
+        <div class="footer-right-col">
           <div class="footer-info">
-            <div class="copyright-text">
-              <small>support@xypo.com</small><br>
-              <small>copyright 2020 ritesh</small>
+            <div class="copyright-text">
```
Comment (saviourcode) [inline index.html:354]: Can you remove this unused div and check once again.

Comment (mahimagoyalx) [reply index.html:354]: > Can you remove this unused div and check once again.

Yes, okay

File: style.css [hunk 2]
```diff
@@ -798,11 +802,12 @@ hr {
...
 }
 .footer-left-col {
-  flex-basis: 50%;
-  margin-top: 60px;
+  flex-basis: 55%;
+  margin-top: 5px;
 }
 .footer-right-col {
-  flex-basis: 35%;
+  flex-basis: 45%;
```
Comment (saviourcode) [inline style.css:809]: Change this to flex-basis: 10%

**Review by sjchoure — CHANGES_REQUESTED** [review]
@mahimagoyalx Please take a note of the changes

**Comment by mahimagoyalx** [comment]
Made changes, check now. @saviourcode

**Review by sjchoure — APPROVED** [review]
LGTM! Awesome work @mahimagoyalx !!

**Comment by mahimagoyalx** [comment]
> LGTM! Awesome work @mahimagoyalx !!

Thank you!

**Comment by mahimagoyalx** [comment]
Will you merge this PR? @saviourcode

**Comment by saviourcode** [comment]
Only our Project Leader @DARK-art108 can do this. I have notified them regarding your PR. He will merge soon.

**Comment by mahimagoyalx** [comment]
> Only our Project Leader @DARK-art108 can do this. I have notified them regarding your PR. He will merge soon.

Okay :)

**Pull request merged**